tar/bsdtar.c

```diff
@@ -1653,12 +1653,6 @@ dooption(struct bsdtar *bsdtar, const char * conf_opt,
 		if (exclude(bsdtar, conf_arg))
 			bsdtar_errc(bsdtar, 1, 0,
 			    "Couldn't exclude %s", conf_arg);
-	} else if (strcmp(conf_opt, "hashes") == 0) {
```

Comment:
When I was reviewing the earlier code, I wondered about adding a `no-hashes` option in case somebody wanted to cancel `hashes` in their config file.  But then I decided that there was no compelling case to support `hashes` in config files, so I've made it solely a command-line option.

I also decided not to bother with `bsdtar->option_hashes_set`, since we normally don't check for "doubly set" options for command-line-only options.  This means that `tarsnap --list-archives --hashes --hashes` is accepted, but I figure that's not a problem.

(We also accept weirdness such as `tarsnap -x --chroot --chroot` or `tarsnap -c --dry-run --dry-run`.)

Reply:
Yeah, I considered that but agree there's no reason to support `hashes` in configuration files and thus no need for `--no-hashes`.